packages/ibm-products/src/components/CoachmarkStack/CoachmarkStack.test.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright IBM Corp. 2023, 2023
```

Comment:
It’s a good thing I’m tenacious. Or maybe it’s not.

```suggestion
 * Copyright IBM Corp. 2023, 2024
```